conf/messages

```diff
@@ -528,3 +528,8 @@ phoneAndTablet=Phone and tablet
 bulletBlitzClassical=Bullet, blitz, classical
 correspondenceChess=Correspondence chess
 onlineAndOfflinePlay=Online and offline play
+gameAnalysis=Game analysis
+followAndChallengeFriends=Follow and challenge friends
+availableInEightyLanguages=Available in 80+ languages!
```

Comment:
@alexbagirov I'm almost certain he's asking for the "80" to also be a separate `%s` that is passed into this (and to change the variable name to something like `availableInEightyToHundredLanguages` in order to keep it relevant into the future).